### PR TITLE
Adding docs for deploying to ElasticBeanstalk

### PR DESCRIPTION
**Pull request by flomotlik**

### CODE REVIEW

**Comment by flomotlik** [comment]
@AlexTi could you take a look at those docs as well, so you're happy with them

File: _posts/docker/deployments/2015-09-01-aws.md [hunk 1]
```diff
@@ -40,12 +40,14 @@ You can get the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` from the IAM set
 
 Before reading through the documentation please take a look at the [Services]({{ site.baseurl }}{% post_url docker/2015-05-25-services %}) documentation page so you have a good understanding how services work and how to include different containers into a specific build.
 
-The `codeship-services.yml` file uses the `codeship/aws-deployment` container and sets the encrypted environment file. To get access to files in the repository that we're running this build for we set up a volume that shares `./` (the repository folder) to `/deploy`. This gives us access to all files in the repository in `/deploy/...` for the following steps.
+The `codeship-services.yml` file uses the `codeship/aws-deployment` container and sets the encrypted environment file. Additionally it sets the `AWS_DEFAULT_REGION` through the environment config setting. To get access to files in the repository that we're running this build for we set up a volume that shares `./` (the repository folder) to `/deploy`. This gives us access to all files in the repository in `/deploy/...` for the following steps.
```
Comment (AlexTi) [inline _posts/docker/deployments/2015-09-01-aws.md]: Added commas:

To get access to files in the repository**,** that we're running this build for**,** we set up a volume that shares `./` (the repository folder) to `/deploy`.

**Comment by AlexTi** [comment]
lgtm 

For future AWS article get already pretty long. We have to think about better structure soon, imo.

File: _posts/docker/deployments/2015-09-01-aws.md (hunk 1 above)
Comment (flomotlik) [inline _posts/docker/deployments/2015-09-01-aws.md]: I don't think those belong there. "that we're running this build for" is not a Nebensatz. But not totally sure. @jschley @ngauthier could you take a look and help us out with comma rules in english?

Comment (ngauthier) [inline _posts/docker/deployments/2015-09-01-aws.md]: That whole section is redundant with the sentence right after it. I would just trim it to:

> The `codeship-services.yml` file uses the `codeship/aws-deployment` container and sets the encrypted environment file. Additionally it sets the `AWS_DEFAULT_REGION` through the environment config setting. We set up a volume that shares `./` (the repository folder) to `/deploy`. This gives us access to all files in the repository in `/deploy/...` for the following steps.

Comment (flomotlik) [inline _posts/docker/deployments/2015-09-01-aws.md]: sounds good, will use your version

**Comment by flomotlik** [comment]
@AlexTi  Agreed, if we do more AWS docs we need to start splitting it off somehow. Possibly a subfolder with separate articles per service

**Pull request merged**